Formula/qt5.rb

```diff
@@ -110,6 +116,11 @@ def install
 
     args << "-skip" << "qtwebengine" if build.without? "webengine"
 
+    if build.with? "qtwebkit"
+      (buildpath/"qtwebkit").install resource("qt-webkit")
+      inreplace ".gitmodules", /.*status = obsolete\n((\s*)project = WebKit\.pro)/, "\\1\n\\2initrepo = true"
```

Comment:
Because nowadays Qt chooses which modules to build based on the `.gitmodules` file.